chore(kosmos): update program name (#1717)
scaleway/docs-content: containers/kubernetes/how-to/edit-kosmos-cluster.mdx

## How to configure external nodes to join the cluster

> 1. Get your server's public IP and SSH to the server :

Reviewer: ```suggestion
1. Get your server's public IP and SSH to the server:
```

> 1. Get your server's public IP and SSH to the server :

Reviewer 2: ```suggestion
1. Get your server's public IP and SSH to the server:
```

    ```bash
    ssh <user>@<server_ip>
    ```

2. Download the node-agent program :

    ```bash
    wget https://scwcontainermulticloud.s3.fr-par.scw.cloud/node-agent_linux_amd64 && chmod +x node-agent_linux_amd64
    ```

> 3. Export the required environment variables :

Reviewer: ```suggestion
3. Export the required environment variables:
```

> 3. Export the required environment variables :

Reviewer 2: ```suggestion
3. Export the required environment variables:
```

    ```bash
    export POOL_ID=<pool_id>  POOL_REGION=<cluster_region>  SCW_SECRET_KEY=<secret_key>
    ```

> 4. Execute the program to attach the node to the multicloud-pool:

Reviewer: ```suggestion
4. Execute the program to attach the node to the multi cloud pool:
```

> 4. Execute the program to attach the node to the multicloud pool :

Reviewer: ```suggestion
4. Execute the program to attach the node to the multicloud pool:
```

> 4. Execute the program to attach the node to the multicloud pool :

Reviewer 2: ```suggestion
4. Execute the program to attach the node to the multicloud-pool:
```
